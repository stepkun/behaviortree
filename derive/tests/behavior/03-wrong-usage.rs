// Copyright Â© 2025 Stephan Kunz

//! Test wrong usage of behavior derive macro `Behavior` 

#[doc(hidden)]
extern crate alloc;

#[derive(behaviortree_derive::Behavior)]
struct TestBehavior;


// dummy main
fn main(){}
