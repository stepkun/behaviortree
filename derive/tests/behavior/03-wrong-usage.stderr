error[E0433]: failed to resolve: could not find `Behavior` in `behaviortree_derive`
 --> tests/behavior/03-wrong-usage.rs:8:31
  |
8 | #[derive(behaviortree_derive::Behavior)]
  |                               ^^^^^^^^ could not find `Behavior` in `behaviortree_derive`
